tests/integration/targets/apache2_module/tasks/actualtest.yml

```diff
@@ -79,69 +64,70 @@
       - 'disablefinal.changed'
 
 - name: set userdir to original state
-  apache2_module:
+  community.general.apache2_module:
     name: userdir
     state: present
   when: userdir_first_disable is changed
 
 - name: ensure autoindex enabled
-  apache2_module:
+  community.general.apache2_module:
     name: autoindex
     state: present
 
 - name: Debian/Ubuntu specific tests
   when: "ansible_os_family == 'Debian'"
   block:
   - name: force disable of autoindex  # bug #2499
-    apache2_module:
+    community.general.apache2_module:
       name: autoindex
       state: absent
       force: True
 
   - name: reenable autoindex
-    apache2_module:
+    community.general.apache2_module:
       name: autoindex
       state: present
 
-  - name: enable evasive module, test https://github.com/ansible/ansible/issues/22635
-    apache2_module:
-      name: evasive
-      state: present
-
+  # mod_evasive is enabled by default upon the installation, so disable first and enable second, to preserve the config
```

Comment:
Why not keep the above task? If tests are run again because something failed in the first run, you will have a problem.

Reply:
It is kept, but I switched places: it had present then absent, now it is absent THEN present. At the very end in main.yml it tests whether the modules at the end are the same as they were at the beginning (I dont think that is really necessary, as the test is destructive anyway, but didn't want to change one more thing in there).